_rules/bd5de3.md

```diff
@@ -0,0 +1,37 @@
+
+---
+name: Focusable element has visible focus
+test_type: atomic
+
+ description: |
+  This rule checks that all elements that can receive focus has a focus styling that is visibly different from the normal styling.
+ success_criterion: 
+- 2.4.7 # Focus Visible
+
+ test_aspects:
+- DOM Tree
+- CSS Styling
+
+ authors:
+- Anne Thyme Nørregaard
+---
+
+## Applicability
+
+This rule applies to all elements that can receive focus.
+
+## Expectation
+
+When the target element's focus state is triggered, a [visible](#visible) change in text style, border, box-shadow, background-image or color occurs for the element.
```

Comment:
I don't think you can use the "visible" definition here. I'm also not sure these are all the CSS properties to look for, but I think this is a good direction. You may want to consider leaving just allowing any style change to indicate focus. Not sure what the best way to do this is.